.eslintrc

```diff
@@ -1,43 +1,231 @@
 {
   "parser": "babel-eslint",
-  "extends": [
-    "plugin:flowtype/recommended",
-    "plugin:react/recommended",
-    "prettier",
-    "prettier/flowtype",
-    "prettier/react"
-  ],
   "plugins": [
-    "flowtype",
     "react",
-    "prettier"
+    "jsx-a11y",
+    "babel",
+    "import"
+  ],
+  "extends":[
+    "plugin:import/errors",
+    "plugin:import/warnings"
   ],
+  "ecmaFeatures": {
+    "modules": true,
+    "jsx": true
+  },
   "parserOptions": {
-    "ecmaVersion": 2016,
+    "ecmaVersion": 8,
     "sourceType": "module",
     "ecmaFeatures": {
-    "jsx": true
+      "jsx": true
     }
   },
   "env": {
+    "browser": true,
     "es6": true,
-    "node": true
+    "node": true,
+    "mocha": true
```

Comment:
We're actually using jest, not mocha.
While they share the "describe" and "it", if we start using more complex jest, this is going to be an issue